pkg/machinery/config/types/v1alpha1/generate/options.go

```diff
@@ -169,6 +196,9 @@ type GenOptions struct {
 	Debug                     bool
 	Persist                   bool
 	MachineDisks              []*v1alpha1.MachineDisk
+	TrustdInfo                *TrustdInfo
```

Comment:
this is totally a nit, but I would probably have this in `Input`, as this doesn't seem to be really an option vs. something we supply as input (secrets), with defaults being to generate new secrets